src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -87,7 +90,11 @@ public virtual async Task ProcessLogQueue(object state)
             while (!_cancellationTokenSource.IsCancellationRequested)
```

Comment:
Question unrelated to your change in this PR. But I am curious.
I see the cancellation token gets cancelled in the Stop method. But that method is not being called by any code path in this repo.
Why is the Start method public? Looks like the only caller is the constructor of this type.